torchgeo/datasets/oscd.py

```diff
@@ -0,0 +1,234 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""OSCD dataset."""
+
+import os
+import glob
+import tifffile
```

Comment:
We don't currently have a dependency on tifffile, can you read the file with rasterio instead?

Reply:
FYI: from my previous benchmarking tifffile is much faster (as I _think_ it goes through libtiff), in places where geographic coordinates aren't a concern we maybe should consider this over rasterio.